pkg/views/profile/list.go

```diff
@@ -5,110 +5,70 @@ package profile
 
 import (
 	"fmt"
-	"os"
 
 	"github.com/daytonaio/daytona/cmd/daytona/config"
 	"github.com/daytonaio/daytona/pkg/views"
 	views_util "github.com/daytonaio/daytona/pkg/views/util"
-
-	"github.com/charmbracelet/lipgloss"
-	"github.com/charmbracelet/lipgloss/table"
-	"golang.org/x/term"
 )
 
-type RowData struct {
+type rowData struct {
 	ID     string
 	Name   string
 	Status string
 	ApiUrl string
 	ApiKey string
 }
 
-func getRowFromRowData(rowData RowData, showApiKeysFlag bool) []string {
-	var state string
-	if rowData.Status == "" {
-		state = views.InactiveStyle.Render("Inactive")
-	} else {
-		state = views.ActiveStyle.Render("Active")
-	}
-
-	row := []string{
-		views.NameStyle.Render(rowData.ID),
-		views.DefaultRowDataStyle.Render(rowData.Name),
-		state,
-		views.DefaultRowDataStyle.Render(rowData.ApiUrl),
-	}
+func ListProfiles(profileList []config.Profile, activeProfileId string, showApiKeysFlag bool) (string, error) {
+	headers := []string{"ID", "Name", "Status", "API URL"}
 	if showApiKeysFlag {
-		row = append(row, views.DefaultRowDataStyle.Render(rowData.ApiKey))
+		headers = append(headers, "API Key")
 	}
 
-	return row
-}
-
-func getRowData(profile *config.Profile, activeProfileId string, showApiKeysFlag bool) *RowData {
-	rowData := RowData{"", "", "", "", ""}
+	data := [][]string{}
 
-	rowData.ID = profile.Id
-	rowData.Name = profile.Name
-	rowData.ApiUrl = profile.Api.Url
-	if profile.Id == activeProfileId {
-		rowData.Status = "1"
-	}
-	if showApiKeysFlag {
-		rowData.ApiKey = profile.Api.Key
+	for _, profile := range profileList {
+		data = append(data, getRowFromData(&profile, activeProfileId, showApiKeysFlag))
 	}
 
-	return &rowData
-}
+	table := views_util.GetTableView(data, headers, nil, func() {
+		fmt.Println(renderUnstyledList(profileList, activeProfileId, showApiKeysFlag))
```

Comment:
Other render function seem to print out by themselves. Let's make it consistent here